scripts/sfpshow

```diff
@@ -321,9 +321,9 @@ class SFPShow(object):
         channel_threshold_align = 18
         module_threshold_align = 15
 
-        if sfp_type.startswith('QSFP'):
+        if sfp_type.startswith('QSFP') or sfp_type.startswith('OSFP'):
             # Channel Monitor
-            if sfp_type.startswith('QSFP-DD'):
+            if sfp_type.startswith('QSFP-DD') or sfp_type.startswith('OSFP'):
```

Comment:
I am thinking we should have all CMIS cables share the same output format, maybe use a more gerneic way?